llvm/include/llvm/CodeGen/TargetFrameLowering.h

```diff
@@ -210,6 +211,11 @@ class TargetFrameLowering {
   /// for noreturn nounwind functions.
   virtual bool enableCalleeSaveSkip(const MachineFunction &MF) const;
 
+  virtual void emitCFIsForCSRsHandledByRA(MachineFunction &MF,
```

Comment:
We need a comment here to tell backend authors what they are supposed to do with this hook and when they should consider implementing it.

Reply:
Thanks @qcolombet I'll make sure I'll add a comment when I create a separate MR for it.